### PR TITLE
Clarify code snippet use

### PR DESCRIPTION
**Pull request by conceptualshark**
## Description

Closes https://github.com/camunda/developer-experience/issues/284

Specifically, I'm trying to make sure each instance of configuration in the backup and restore docs appropriately describes how the user could actually make that config change/what file that change should occur in.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by christinaausley** [comment]
Hi @conceptualshark 👋 Are you still making changes here, or should this issue be moved to `Ready`, or are you ready for review?

**Comment by conceptualshark** [comment]
Sorry for the wait on this one, it should be ready to go!

File: docs/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Even when the underlying storage bucket is the same, backups from one are not co
 
 ### S3 backup store
 
-To store your backups in any S3 compatible storage system such as [AWS S3] or [MinIO], set the backup store to `S3` and tell Zeebe how to connect to your bucket:
+To store your backups in any S3 compatible storage system such as [AWS S3] or [MinIO], set the backup store to `S3` and tell Zeebe how to connect to your bucket. This configuration can be set in your Zeebe [`config/application.yaml`](/docs//self-managed/zeebe-deployment/configuration/configuration.md):
```
Comment (aabouzaid) [inline docs/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md:29]: Is the double slash intended? `/docs//`

**Review by aabouzaid — APPROVED** [review]

File: docs/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md (hunk 1 above)
Comment (conceptualshark) [reply docs/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md]: Thanks for catching this. updated to fix 👍

File: docs/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md [hunk 2]
```diff
@@ -37,33 +40,68 @@ The backup API can be reached via the Actuator management port, which by default
...
+</TabItem>
+</Tabs>
 
-for Tasklist:
+#### Tasklist
+
+<Tabs groupId="config" className="tabs-hidden" defaultValue="yaml" values={
+[
+{label: 'YAML file', value: 'yaml', },
+{label: 'Environment Variable', value: 'env', },
```
Comment (christinaausley) [inline docs/self-managed/operational-guides/backup-restore/operate-tasklist-backup.md:81]: Do we mean "Environment variable" for sentence case here?

**Review by christinaausley — APPROVED** [review]

**Review by aabouzaid — APPROVED** [review]

**Pull request merged**